test/e2e/e2e_test.go

```diff
@@ -0,0 +1,20 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	"k8s.io/kubernetes/test/e2e"
+	"testing"
```

Comment:
nit, golang style is to separate the stdlib imports from third party with a line break